src/components/SelectionList/BaseListItem.tsx

```diff
@@ -23,6 +24,7 @@ function BaseListItem<TItem extends ListItem>({
     onCheckboxPress,
     onDismissError = () => {},
     rightHandSideComponent,
+    checkmarkPosition = 'left',
```

Comment:
WE have consts for these
```suggestion
    checkmarkPosition = CONST.DIRECTION.LEFT,
```